### ISSUE TITLE
The unsatisfactory results under several scenes of Tank&Temples.

### ISSUE THREAD

**Issue by Apiecehann** [ubc-vision/3dgs-mcmc#31]
Firstly, I really appreciate your work. Recently, I've tried to reproduce 3DGS-MCMC in the Tank&Temples dataset provided by PixelGs. However, I found that it has relatively low PSNR (Peak Signal-to-Noise Ratio), approximately around 12 to 14, on five subdatasets, namely Auditorium, Courthouse, Lighthouse, Palace and Temple. I've also attempted to export several test set images of the Auditorium.
![b11d1fc5-5c21-4de9-802f-25c490c7f31d](https://github.com/user-attachments/assets/41b0d05c-1ac1-4054-a3dd-7097655cef00)
Since you didn't provide the config files of Tank&Temples in other scenes expect train and truck, I used the truck.json file as my parameter input. I'm wondering if there's something wrong with my command settings, which has led to the abnormal scene fitting effect. The command I used is:
python train.py --source_path /root/autodl-tmp/data/tnt/Auditorium --eval --config /root/autodl-tmp/code/3dgs-mcmc/configs/truck.json -m ./eval/Auditorium --init_type sfm
python render.py --source_path /root/autodl-tmp/data/tnt/Auditorium -m ./eval/Auditorium --skip_train
python metrics.py -m ./eval/Auditorium

python train.py --source_path /root/autodl-tmp/data/tnt/Courthouse --config /root/autodl-tmp/code/3dgs-mcmc/configs/truck.json --eval -m ./eval/Courthouse --init_type sfm
python render.py --source_path /root/autodl-tmp/data/tnt/Courthouse -m ./eval/Courthouse --skip_train
python metrics.py -m ./eval/Courthouse

python train.py --source_path /root/autodl-tmp/data/tnt/Lighthouse --config /root/autodl-tmp/code/3dgs-mcmc/configs/truck.json --eval -m ./eval/Lighthouse --init_type sfm
python render.py --source_path /root/autodl-tmp/data/tnt/Lighthouse -m ./eval/Lighthouse --skip_train
python metrics.py -m ./eval/Lighthouse

python train.py --source_path /root/autodl-tmp/data/tnt/Palace --config /root/autodl-tmp/code/3dgs-mcmc/configs/truck.json --eval -m ./eval/Palace --init_type sfm
python render.py --source_path /root/autodl-tmp/data/tnt/Palace -m ./eval/Palace --skip_train
python metrics.py -m ./eval/Palace

python train.py --source_path /root/autodl-tmp/data/tnt/Temple --config /root/autodl-tmp/code/3dgs-mcmc/configs/truck.json --eval -m ./eval/Temple --init_type sfm
python render.py --source_path /root/autodl-tmp/data/tnt/Temple -m ./eval/Temple --skip_train
python metrics.py -m ./eval/Temple

The link to Tank&Temples provided by PixelGs is: https://connecthkuhk-my.sharepoint.com/personal/u3009782_connect_hku_hk/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fu3009782%5Fconnect%5Fhku%5Fhk%2FDocuments%2Ftanks%5Fand%5Ftemples%2Ezip&parent=%2Fpersonal%2Fu3009782%5Fconnect%5Fhku%5Fhk%2FDocuments&ga=1

Thank you very much for your reply。

**Comment by Apiecehann** [comment]
Thanks for your Work! I found that when I removed the terms args.opacity_reg * torch.abs(gaussians.get_opacity).mean() and args.scale_reg * torch.abs(gaussians.get_scaling).mean() in the loss item and only used color as the loss, it could converge and achieve a relatively good result.

**Comment by deivse** [comment]
For anyone else who might encounter this issue in the future, the final effect is uncontrollable growth of the scale of Gaussians in the scene, resulting in rapidly deacreasing eval metrics as well as training loss. I'm not as of now sure of the exact mechanism, but in my testing (same issue, but on scannet++), the **fix is to lower/disable opacity regularization**. Hopefully I'll have time to figure out what the exact mechanism for this is.

**Comment by Apiecehann** [comment]
Thanks for your applying.
